lib/fastlane/fast_file.rb

```diff
@@ -202,14 +203,14 @@ def import_from_git(url: nil, path: 'fastlane/Fastfile')
           Actions.sh("git clone '#{url}' '#{folder}' --depth 1 -n")
```

Comment:
I just tried it with my setup and it appears that this line may cause problem.
For the 'cloning' case, adding `--branch #{branch}` solve the problem.
However, if the user want to specify another branch later, the `--depth 1` will prevent him from doing it because only changes made to the cloned branch will be fetched.